src/goto-programs/remove_function_pointers.cpp

```diff
@@ -14,12 +14,14 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <util/config.h>
 #include <util/std_expr.h>
 #include <util/type_eq.h>
+#include <util/message.h>
 
 #include <ansi-c/c_types.h>
 
 #include "remove_skip.h"
 #include "remove_function_pointers.h"
 #include "compute_called_functions.h"
+#include <goto-programs/remove_const_function_pointers.h>
```

Comment:
Use `#include "remove_const_function_pointers.h"` for consistency with the preceding ones.